config/team_config.json

```diff
@@ -0,0 +1,18 @@
+[
+    {
+        "id": "1",
+        "name": "Formula Student Team Delft",
+        "master": "http://localhost:11311 ",
+        "car_settings": {
+            
+        }
+    },
+    {
+        "id": "2",
+        "name": "TUFast",
+        "master": "http://localhost:11312 ",
+        "car_settings": {
+
+        }
+    }
+]
```

Comment:
The team selector buttons are created from this config file. Could be nice to include teams car settings here as well?

Reply:
YES!

Reply:
yeah maybe some sort of path to their settings.json file?